### PR TITLE
feat: move to arm64 architecture

### PR DESCRIPTION
**Pull request by samchungy**
Gantry now has support for arm64. We can pretty much move our builds to arm64.

Todo: Bump up gantry version once the hotfix lands.

### CODE REVIEW

File: template/express-rest-api/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ RUN yarn build
 
 ###
 
-FROM gcr.io/distroless/nodejs:16 AS runtime
+FROM gcr.io/distroless/nodejs:16-arm64 AS runtime
```
Comment (samchungy) [inline template/express-rest-api/Dockerfile:20]: not sure if FROM --platform=linux/arm64 works here. I'll have to test

**Comment by 72636c** [comment]
One complication with early adoption here is that existing teams will be using Intel build agents. We'd have to figure out how to best communicate that they should provision a Graviton cluster for their new project.

**Comment by samchungy** [comment]
> One complication with early adoption here is that existing teams will be using Intel build agents. We'd have to figure out how to best communicate that they should provision a Graviton cluster for their new project.

yeah I noticed gantry had some sections on this actually

**Comment by 72636c** [comment]
One option may be to scrap the queue templating and just inline a bunch of documentation/links/TODOs in our `.buildkite/pipeline.yml` template files

https://github.com/seek-oss/skuba/blob/034fe46183e58d5d8fda54bec60fecb27e05342a/template/koa-rest-api/skuba.template.js#L13-L24

**Comment by samchungy** [comment]
Our buildAgents only support x86 for the `GET_GITHUB_TOKEN` step for now. Will wait til support is added:

https://seekchat.slack.com/archives/C01J8F3LELB/p1653619527077269?thread_ts=1653617669.275119&cid=C01J8F3LELB

**Review by 72636c — APPROVED** [review]

**Pull request merged**